src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -48,7 +52,8 @@ export class AuthenticationApiImpl implements AuthenticationApi {
         'g-recaptcha-response': recaptcha,
         'firstName': firstName,
         'lastName': lastName,
-        'from': email,
+        'from': email ?? mobilePhone,
```

Comment:
I agree : However, this is managed by the MSG-GTW and I can't change it easily. We'll need to refactor it and pass through all processes as well, which won't be trivial, as it will "break" the registration processes during the operation